ouroboros-consensus-cardano/src/cardano-testlib/Test/Consensus/Cardano/ProtocolInfo.hs

```diff
@@ -0,0 +1,357 @@
+{-# LANGUAGE DataKinds           #-}
+{-# LANGUAGE FlexibleContexts    #-}
+{-# LANGUAGE GADTs               #-}
+{-# LANGUAGE NamedFieldPuns      #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications    #-}
+
+-- | Utility functions to elaborate a Cardano 'ProtocolInfo' from certain parameters.
+module Test.Consensus.Cardano.ProtocolInfo (
```

Comment:
The section in the export list doesn't match the sectioning in the file

Reply:
Thanks. Changed!